src/scenario.cpp

```diff
@@ -647,6 +644,15 @@ char *clean_cdata(char *ptr, int *removed_crlf = NULL)
         memmove(ptr, ptr + 1, strlen(ptr));
     }
 
+    /* Re-add our removed (CR)LFs. */
+    while (removed_crlf--) {
+        strcat(msg, "\n");
+    }
+    /* A SIP message must contain a double (CR)LF to signify EOH. */
```

Comment:
I think it must actually contain a double CRLF - the CR is mandatory. Worth a comment to say that the \r gets added by the SendingMessage constructor so we don't need it here.